backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/process/view/ViewActionDescriptor.java

```diff
@@ -102,6 +99,14 @@ public String getActionId()
 		return parametersDescriptor.build();
 	}
 
+	public void addParametersDescriptor(final DocumentEntityDescriptor.Builder parametersDescriptor)
+	{
+		viewActionParamDescriptors.stream()
+				.filter(ViewActionParamDescriptor::isUserParameter)
+				.map(ViewActionParamDescriptor::createParameterFieldDescriptor)
+				.forEach(parametersDescriptor::addField);
+	}
+
```

Comment:
I think this is no longer used, right? aka, can be reverted.

Reply:
Still used